### ISSUE TITLE
Handle entity velocities.

### ISSUE THREAD

**Issue by t81lal** [DarkStorm652/DarkBot#26]
https://github.com/DarkStorm652/DarkBot/blob/ca22d51ac8124efe39a3463ac5ca613e32da4678/src/main/java/org/darkstorm/darkbot/minecraftbot/world/BasicWorld.java
@EventHandler
    public void onLivingEntitySpawn(LivingEntitySpawnEvent event)

what happens to the spawnmob velocity data?

**Comment by DarkStorm652** [comment]
Entity velocity in general is currently not handled.

**Comment by t81lal** [comment]
only way to fix would be to add do a watched list and loop every game tick and then add to the position :/

**Comment by DarkStorm652** [comment]
Yep, I just figured that the server would update their positions often enough for it not to matter (and it does, but it might as well be implemented anyway).
